buildspec.yml

```diff
@@ -0,0 +1,10 @@
+version: 0.2
+
+env:
+    parameter-store:
+        OAUTH: "OAUTH_TOKEN"
+
+phases:
+    build:
+        commands:
+            - script/build
```

Comment:
Don't forget to check this is executable

Reply:
I forgot.